pages/index.js

```diff
@@ -141,11 +167,16 @@ const Home = ({ latestStories }) => {
 
   const [caseData, setCaseData] = useState({})
   const [cumulativeCase, setCumulativeCase] = useState(null)
+  const [cumulativeTest, setCumulativeTest] = useState(null)
 
   const [cumulativeData, setCumulativeData] = useState({})
   const [totalCases, setTotalCases] = useState(null)
   const [totalCasesDate, setTotalCasesDate] = useState(null)
   
+  const commaNumber = (number) => {
+    if (number == null) {return "0"}
+    return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
prefer String() instead of .toString, more defined behavior in cases where input object doesn't have Object.prototype.toString defined.